neo4j-cli/aura/internal/subcommands/import/job/get_test.go

```diff
@@ -0,0 +1,564 @@
+                        "created_indexes": 0,
+                        "created_relationships": 33121,
+                        "id": "r:0",
+                        "processed_rows": 33121,
+                        "total_rows": 33121,
+                        "type": "FLIGHT_USES_AIRCRAFT_DATA"
+                    },
+                    {
+                        "created_constraints": 0,
+                        "created_indexes": 0,
+                        "created_relationships": 33121,
+                        "id": "r:3",
+                        "processed_rows": 33121,
+                        "total_rows": 33121,
+                        "type": "FLIGHT_USES_AIRCRAFT"
+                    },
+                    {
+                        "created_constraints": 0,
+                        "created_indexes": 0,
+                        "created_relationships": 33121,
+                        "id": "r:2",
+                        "processed_rows": 33121,
+                        "total_rows": 33121,
+                        "type": "FLIGHT_ARRIVES_AT_AIRPORT"
+                    },
+                    {
+                        "created_constraints": 0,
+                        "created_indexes": 0,
+                        "created_relationships": 33121,
+                        "id": "r:5",
+                        "processed_rows": 33121,
+                        "total_rows": 33121,
+                        "type": "FLIGHT_VIEW_ARRIVES_AT_AIRPORT"
+                    },
+                    {
+                        "created_constraints": 0,
+                        "created_indexes": 0,
+                        "created_relationships": 33121,
+                        "id": "r:4",
+                        "processed_rows": 33121,
+                        "total_rows": 33121,
+                        "type": "FLIGHT_VIEW_DEPARTS_FROM_AIRPORT"
+                    },
+                    {
+                        "created_constraints": 0,
+                        "created_indexes": 0,
+                        "created_relationships": 1045726,
+                        "id": "r:7",
+                        "processed_rows": 1045726,
+                        "total_rows": 1045726,
+                        "type": "FLIGHT_HAS_TICKET"
+                    },
+                    {
+                        "created_constraints": 0,
+                        "created_indexes": 0,
+                        "created_relationships": 33121,
+                        "id": "r:6",
+                        "processed_rows": 33121,
+                        "total_rows": 33121,
+                        "type": "FLIGHT_VIEW_USES_AIRCRAFT"
+                    }
+                ]
+            },
+            "start_time": "2025-08-15T13:12:51Z",
+            "state": "Completed",
+            "submitted_time": "2025-08-15T13:12:50.499797Z"
+        },
+        "user_id": "f607bebe-0cc0-4166-b60c-b4eed69ee7ee"
+    }
+}
+	`)
```

Comment:
There is some custom logic to display progress tables, it is probably worth it adding an additional test to confirm this displays correctly (similar to `dataapi/graphql/create_test.go`)